aggregations/db_tables/deployed_contracts.py

```diff
@@ -60,3 +81,102 @@ def start_of_range(self, timestamp: int) -> int:
     @staticmethod
     def prepare_data(parameters: list, *, start_of_range=None, **kwargs) -> list:
         return parameters
+
+    def store(self, parameters: list) -> list:
+        super().store(parameters)
+
+        near_rpc_url = os.getenv("NEAR_RPC_URL")
+        if not near_rpc_url:
+            return
+        near_rpc = near_api.providers.JsonProvider(near_rpc_url)
+
+        def view_code(account_id, block_id):
+            for _ in range(1000):
```

Comment:
Are you sure we want to try it 1000 times?

Reply:
Yes, I would prefer to avoid unnecessary crashes of analytics. JSON RPC could be flaky.